src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -3,20 +3,23 @@
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 import edu.wpi.first.wpilibj.Encoder;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
-public class Drive {
+public class Drive extends SubsystemBase {
 
   public Encoder leftEncoder;
   public Encoder rightEncoder;
 
   private Gearbox leftGearbox;
   private Gearbox rightGearbox;
   // left gear box CAN ids
-  private final int LEFT_BACK_CAN_ID = 6;
-  private final int LEFT_FRONT_CAN_ID = 12;
+  private final int LEFT_BACK_CAN_ID = 6;//6
+  private final int LEFT_FRONT_CAN_ID = 5;//12
   // right gear box CAN ids
-  private final int RIGHT_BACK_CAN_ID = 4;
-  private final int RIGHT_FRONT_CAN_ID = 5;
+  private final int RIGHT_BACK_CAN_ID = 8;//4
```

Comment:
Right is now:

  private final int RIGHT_BACK_CAN_ID = 4;
  private final int RIGHT_FRONT_CAN_ID = 5;
  //  private final int RIGHT_MIDDLE_CAN_ID = 6;